CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 * [CHANGE] Redo tempo-cli with basic command structure and improvements [#385](https://github.com/grafana/tempo/pull/385)
 * [CHANGE] Add content negotiation support and sharding parameters to Querier [#375](https://github.com/grafana/tempo/pull/375)
 * [ENHANCEMENT] Add docker-compose example for GCS along with new backend options [#397](https://github.com/grafana/tempo/pull/397)
+* [ENHANCEMENT] tempo-cli list blocks usability improvements [#403](https://github.com/grafana/tempo/pull/403)
```

Comment:
```suggestion
* [ENHANCEMENT] Tempo CLI list blocks usability improvements [#403](https://github.com/grafana/tempo/pull/403)
```

Reply:
In this case `tempo-cli` is the executable name and would like to keep it for consistency.